docs/docs/run-node/04-rosetta.md

```diff
@@ -77,7 +77,7 @@ import (
 
 "context"
 "github.com/coinbase/rosetta-sdk-go/types"
-"github.com/cosmos/cosmos-sdk/server/rosetta/lib"
+"github.com/cosmos/cosmos-sdk/rosetta/lib"
```

Comment:
Can you update this?

Reply:
done!

> Can you have a look at the last comments and the tests? Then it's ready to be merged :)

done with the comments, taking a look to the tests now